src/tracer.js

```diff
@@ -80,6 +80,9 @@ export default class Tracer {
             } else {
                 fields.operationName = nameOrFields;
             }
+            if (fields.parent) {
+                fields.parent = fields.parent._imp;
```

Comment:
Of course we can do this in implementation, but the implementation should always receive arguments it implement